backend/app/controllers/spree/admin/resource_controller.rb

```diff
@@ -158,7 +158,8 @@ def parent_data
   def parent
     if parent_data.present?
       @parent ||= parent_data[:model_class].
-          send("find_by_#{parent_data[:find_by]}", params["#{resource.model_name}_id"])
+        # Don't use find_by to workaround globalize/globalize#441 bug
+        send("find_by", parent_data[:find_by].to_s => params["#{resource.model_name}_id"])
```

Comment:
Hi i fixed the comment and passed the first argument to send as symbol as requested. Let me know if it is ok now. Thanks.